portal/views/patient.py

```diff
@@ -472,7 +472,8 @@ def patient_timewarp(patient_id, days):
     from portal.models.questionnaire_response import QuestionnaireResponse
     from portal.models.user_consent import UserConsent
 
-    if current_app.config['SYSTEM_TYPE'] not in ('development', 'testing'):
+    if current_app.config['SYSTEM_TYPE'] not in (
+            'demo', 'development', 'testing'):
```

Comment:
Why not just "not production"?
```suggestion
    if current_app.config['SYSTEM_TYPE'] != "production":
```